internal/models/models.go

```diff
@@ -13,7 +13,9 @@ type Log struct {
 
 type Publisher struct {
 	gorm.Model
-	Name string `json:"name"`
+	OrganizationID string `json:"organizationId"`
+	URL            string `json:"url"`
+	Email          string `json:"email"`
```

Comment:
I'd leave out the email, we don't really use it

Reply:
So in that case we will save only ID and URL right?

Reply:
I'm not convinced about leaving out email field. Being a PEC this could be used to rebuild all steps for the onboarding process for a given publisher. wdyt?